pkg/eve/runme.sh

```diff
@@ -312,5 +346,7 @@ if [ "$(ls -A /in)" ]; then
    mcopy -o -i /bits/config.img -s /in/* ::/
 fi
 
+echo "Generating image for platform '$PLATFORM'..."
```

Comment:
For some reason, this line becomes an internal image name that EVE sees. Later, it mismatches with the expected image name, so the update fails:
```
2025-07-13 21:05:59.778 /pillar/cmd/zedagent/reportinfo.go:760: Device swInfo: partitionLabel:"IMGA" partitionDevice:"/dev/sda2" partitionState:"unused" status:DELIVERED shortVersion:"Generating image for platform 'generic'...\r\n0.0.0-pr4960-c73f4c77-kvm-amd64" swErr:{description:"checkInstalledVersion: image name not match. config Generating image for platform 'generic'...\r\n0.0.0-pr4960-c73f4c77-kvm-amd64, image ver 0.0.0-pr4960-c73f4c77-kvm-amd64\n" timestamp:{seconds:1752439125 nanos:394827219} severity:SEVERITY_ERROR} downloadProgress:100
```

I also dropped a message regarding it on Slack.

Reply:
@OhmSpectator here is what's happening: stream redirection is not working from docker container even thoug we have `exec 3 >&2` it just doens't work. I can (and I will) remove this line but the problem is still there for images like installer. they are prepended by many stdout/stderr messages, this i swhy now we use `-v$(PWD):/out` when generating images from container